bazel/coverage/collect_cc_coverage.sh

```diff
@@ -97,25 +99,27 @@ function llvm_coverage() {
 # - output_file     The location of the file where the generated code coverage
 #                   report is written.
 function gcov_coverage() {
-  local output_file="${1}"; shift
+  local gcda gcno_path line output_file="${1}"
+  shift
 
   # We'll save the standard output of each the gcov command in this log.
-  local gcov_log="$output_file.gcov.log"
+  # this appears to be unused
+  # local gcov_log="$output_file.gcov.log"
```

Comment:
@zuercher im wondering whether its better to remove this or add `> "$gcov_log"` to the end of the `gcov` command. 

afaict, at least in the ci pipeline its not going to be collected by anything

Reply:
...and probs would suppress stdout too

Reply:
I think just remove it.